seedfarmer/commands/_stack_commands.py

```diff
@@ -109,10 +109,17 @@ def destroy_managed_policy_stack(account_id: str, region: str) -> None:
         _logger.info(
             "Destroying Stack %s in Account/Region: %s/%s", info.PROJECT_MANAGED_POLICY_CFN_NAME, account_id, region
         )
-        services.cfn.destroy_stack(
-            stack_name=info.PROJECT_MANAGED_POLICY_CFN_NAME,
-            session=session,
-        )
+        import botocore.exceptions
+
+        try:
+            services.cfn.destroy_stack(
+                stack_name=info.PROJECT_MANAGED_POLICY_CFN_NAME,
+                session=session,
+            )
+        except (botocore.exceptions.WaiterError, botocore.exceptions.ClientError):
+            _logger.info(
```

Comment:
what are the side effects of ignoring the stack destroy error?

Reply:
The stack remains in in CFN.  When then next `seedfarmer apply` is issued, the stack corrects itself.  

NOTE: if there are no more deployments and this error occurs, then that indicates there was a role that was not properly deleted (ex. the CLI session timed out when destroying a deployment that takes longer than the cli session timeout)